### DISCUSSION TITLE
Choosing a different channel for a cask/formula

### DISCUSSION THREAD

**Discussion by eatstorming** [Homebrew/discussions#3848] [Casks]
My question is related to BetterTouchTool, but I believe the underlying point is broader than that.

I recently noticed that BTT is on the Alpha updates channel on Homebrew, and I found that out when a couple of weeks ago an update [triggered a bug with it](https://community.folivora.ai/t/macos-menubar-broken-black-with-widget-mode-test/28753). I had noticed BTT's high update frequency before, but I just thought the devs were trigger-happy with updates.

I then tried finding a way to change the install on my machine to follow the stable version, but could not find it. I found that I could "pin" it with `brew pin`, but my understanding is that this would only stop it from being updated until I do a `brew unpin` it. I also cannot see different versions for it like there are, for example, brave-browser, brave-browser-beta, brave-browser-dev and brave-browser-nightly; there is only bettertouchtool and that's following the alpha channel for the app.

Is there a way to set it to follow the regular/stable channel instead? I don't want to stop it from getting updates altogether, but I also don't want an alpha/beta/dev version that is likely to introduce bugs or instability.

**Comment by SMillerDev** [comment]
Generally the Homebrew/cask tap should have the stable versions. If it is instead tracking alpha releases then a pull request to rectify that would be very welcome. As for your example of brave, the nightly etc are in Homebrew/cask-versions

**Comment by eatstorming** [comment]
It looks like a [deliberate decision](https://github.com/Homebrew/homebrew-cask/issues?q=bettertouchtool) in this case. I found some bug reports about BTT being outdated on Homebrew, but I could not find a discussion on why the decision to follow the alpha channel was made. At any rate, I don't feel like doing PRs to rectify it every time it is updated to the alpha version, nor am I inclined to maintaining a new stable formula for it.

I'll uninstall it from brew and manually install a stable version.

Thank you.
